model/model.go

```diff
@@ -66,6 +66,11 @@ func (err JSONError) Error() string {
 	return err.Detail
 }
 
+// Dataset holds information about entire dataset
+type Dataset struct {
```

Comment:
What is `Dataset`? It seems like a generic name for a specific struct.

Should it be named something clearer?

Reply:
Alternatively, should it be made into an anonymous struct? or just leave the data type as the array of nameservers?

Reply:
I had also been considering Summary but that seemed even more generic.  I welcome your advice on any alternate data structures; I was trying to model it on the existing patterns you had used for putting data into the templates.  I imagined that in the future we might want several fields that describe the entire dataset, whereas the other objects describe individual pieces of data.